### DISCUSSION TITLE
Grab Rate isn't fast enough to allow damage

### DISCUSSION THREAD

**Discussion by finytes** [CosmicHikari/Weather2-Remastered#59] [Questions] [answered]
As the title suggests, grab rates aren't high enough to cause any damage to buildings. No idea if this is our grab list/wind resistance (or if we're even using them correctly), but would appreciate some insight on this if possible. Do note I have tried turning up various integer values in the grab config file to no avail.

**Comment by CosmicHikari** [comment]
Do you have use grablist enabled? @Mrbt0907 talked about the grablist changes while he was working on the update, I don't remember the exact settings required outside of enabling the grab list.

Edit: I believe there is a conditional mode too, but I could be wrong. Will load up my game and have a look at the options now.

**Comment by CosmicHikari** [comment]
Okay so update. 
![image](https://user-images.githubusercontent.com/16886014/171775023-9ea8101f-2a90-449e-aac1-7ffd4e46bb09.png)


* Grab List Entries should be a list of entries separated by a space.

* Grab List Partial Matches can be set to true or false. If true, anything that matches a given entry (Example: stone), cobblestone, sand stone, stone, stone bricks, etc. would all be grabbed. This is good if you have modded leaves from things like Pam's Harvestcraft, as anything containing "leaves" would be picked up.

* Grab list strength match - Set to true if you want stronger tornadoes (EF3-EF5) to do more damage, while weaker ones (EF0-EF2) do less damage. If you want a tornado to always do damage regardless of its strength (EF0 - EF5 does all the same damage) then set this to false.

Please play around with these and update us on what happens 😄 


Edit: clarifications

**Comment by finytes** [comment, marked as answer]
Finally got it, turning up the config option 'Storm_Tornado_maxFlyingEntityBlocks' worked, assuming it was limiting the amount of blocks being picked up because of the amount of blocks already in the air. Thanks for the help!

**Reply by CosmicHikari** [reply]
Ah yes - I forgot all about that. I believe the amount of flying entities is conservative and impacts your FPS / RAM usage. Glad you figured it out!
